lib/ui/views/projects/project_preview_fullscreen_view.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+import 'package:photo_view/photo_view.dart';
+import 'package:transparent_image/transparent_image.dart';
+
+class ProjectPreviewFullScreen extends StatelessWidget {
+  final String projectImage;
+  const ProjectPreviewFullScreen({Key key, this.projectImage})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Project Preview'),
```

Comment:
It would be better if we show title of the project instead of Project Preview here.

Reply:
hi, I have made all the above changes just adding test is left should I write the test before committing?

Reply:
If you are comfortable to write test then yes otherwise you can start writing tests and complete missing tests later in another PRs. `ProjectPreviewFullScreen` requires 1 UI test which is not that important.